### PR TITLE
Add notes on SDK and linker options

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnetcore/blazor/host-and-deploy/configure-linker.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 title: Configure the Linker for ASP.NET Core Blazor
 author: guardrex
 description: Learn how to control the Intermediate Language (IL) Linker when building a Blazor app.
-monikerRange: '>= aspnetcore-3.1'
+monikerRange: '>= aspnetcore-3.1 && <=aspnetcore-5.0'
```
Comment (pranavkm) [inline aspnetcore/blazor/host-and-deploy/configure-linker.md:5]: Most of the content for the 3.2 linker does not apply to 5.0. I wasn't sure if I could have two pages with the same uid

File: aspnetcore/blazor/host-and-deploy/configure-linker.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 ms.author: riande
 ms.custom: mvc
 ms.date: 05/19/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
-uid: blazor/host-and-deploy/configure-linker
+uid: blazor/host-and-deploy/configure-trimmer
```
Comment (pranavkm) [inline aspnetcore/blazor/host-and-deploy/configure-linker.md:10]: The runtime team made a conscious decision to call this trimmer. How do we feel about breaking bookmarks?

File: aspnetcore/blazor/host-and-deploy/configure-linker.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/blazor/host-and-deploy/configure-linker.md:5]: I have a feeling this is going to break the doc build system, but it's never been tried before. I'll look at the build report and see how it went.

File: aspnetcore/blazor/host-and-deploy/configure-linker.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/blazor/host-and-deploy/configure-linker.md:10]: I'll need to do perhaps a touch of work on that. Rick prefers stability in physical locations (URLs) and UIDs. I prefer going ahead and changing them. There are pros and cons of both approaches. Not changing helps to preserve history and requires fewer changes ... but not changing makes a mess of a repo with topics that have virtually irrelevant UIDs and physical locations (and URLs) that don't match where they lay in the ToC. So ...... it's not a bed of roses either way. :smile:

My hours are burned for the week ... is it ok to pick this up on Monday morning?

File: aspnetcore/blazor/host-and-deploy/configure-linker.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/blazor/host-and-deploy/configure-linker.md:5]: Yeah .... thought so ....

# :boom: :cry:

> moniker-overlapping ... Two  or more documents with the same uid  `blazor/host-and-deploy/configure-trimmer`('blazor/host-and-deploy/configure-linker.md',  'blazor/host-and-deploy/configure-trimmer.md') have defined overlapping  moniker: 'aspnetcore-5.0'.

I'll create a new PR that does this on Monday, then I'll ping u to review it to make sure that it's all good.

**Comment by guardrex** [comment]
Moved to :point_right: https://github.com/dotnet/AspNetCore.Docs/pull/19849

**Pull request closed without merging**